specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-02-15-preview/share.json

```diff
@@ -0,0 +1,2019 @@
+        "operationId": "SentShares_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/referenceName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/skipToken"
+          },
+          {
+            "$ref": "#/parameters/filter"
+          },
+          {
+            "$ref": "#/parameters/orderby"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SentShareList"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SentShares_List": {
+            "$ref": "./examples/SentShares_List.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/sentShares/{sentShareId}": {
+      "get": {
+        "tags": [
+          "SentShare"
+        ],
+        "summary": "Get a sent share by guid.",
+        "description": "Get a sent share",
+        "operationId": "SentShares_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/sentShareId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SentShare"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SentShares_Get": {
+            "$ref": "./examples/SentShares_Get.json"
+          },
+          "SentShares_GetAdlsGen2Account": {
+            "$ref": "./examples/SentShares_GetAdlsGen2Account.json"
+          },
+          "SentShares_GetBlobAccount": {
+            "$ref": "./examples/SentShares_GetBlobAccount.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "SentShare"
+        ],
+        "summary": "Create or replace a sent share.",
+        "description": "Create a sent share",
+        "operationId": "SentShares_Create",
```

Comment:
This is confusing. Does this operation _only_ create or can it also replace? Please make the summary, description, and operationId consistent. And if replace is supported, add a 200 response.

Reply:
In case of a replace, it might be long running as well. Is it ok to return a 200 with a long running operation header?

Reply:
Yes.